src/cow-react/modules/limitOrders/hooks/useTradeFlowContext.ts

```diff
@@ -0,0 +1,69 @@
+import { TradeFlowContext } from 'cow-react/modules/limitOrders/services/tradeFlow'
+import { useWeb3React } from '@web3-react/core'
+import { OrderKind } from '@cowprotocol/contracts'
+import { useLimitOrdersTradeState } from 'cow-react/modules/limitOrders/hooks/useLimitOrdersTradeState'
+import { CurrencyAmount, Percent } from '@uniswap/sdk-core'
+import { Token } from '@uniswap/sdk-core'
+import { useWalletInfo } from 'hooks/useWalletInfo'
+import { useGP2SettlementContract } from 'hooks/useContract'
+import { useDispatch } from 'react-redux'
+import { AppDispatch } from 'state'
+import { useAppData } from 'hooks/useAppData'
+
+// TODO: set proper values
+const deadline = 5 // 5 min
+const allowedSlippage = new Percent(0)
```

Comment:
I think in the limit order context, slippage should be fixed to `0` always.

Reply:
Yes, we don't send anything with slippage. The limit price is the minimun price we accept